Buscorepuestos/buscorepuestos-web
# Feature/br 455 products

## What type of PR is this? (check all applicable)


## Description
Crear la pagina de productos y conectar el front a la api para mostrar los productos de Sideo y FG Automotion siguiendo el diseño propuesto.

## Related Tickets & Documents
https://linear.app/buscorepuestos/issue/BR-455/products

## QA Instructions, Screenshots, Recordings

1. Ejecutar el front en local y abrir la pagina de la [`tienda`](http://localhost:4000/store).
2. Verificar la correspondencia con el diseño y el responsive.

Also make sure you check this browser compatibility version for desktop and mobile:

## Checklist

DO NOT send the PR without completing the checklist above:


## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

      have not been included_

## Are there any post deployment tasks we need to perform?
